src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -94,17 +94,7 @@ public TracerSettings(IConfigurationSource source)
 
             var maxTracesPerSecond = source?.GetInt32(ConfigurationKeys.MaxTracesSubmittedPerSecond);
 
-            if (maxTracesPerSecond != null)
-            {
-                // Ensure our flag for the rate limiter is enabled
-                RuleBasedSampler.OptInTracingWithoutLimits();
-            }
-            else
-            {
-                maxTracesPerSecond = 100; // default
-            }
-
-            MaxTracesSubmittedPerSecond = maxTracesPerSecond.Value;
+            MaxTracesSubmittedPerSecond = maxTracesPerSecond ?? 100;
```

Comment:
Just for consistency, can you combine this line with the previous line, like the other settings above?
(github wouldn't let me suggest)

```csharp
MaxTracesSubmittedPerSecond = source?.GetInt32(ConfigurationKeys.MaxTracesSubmittedPerSecond) ??
                              // default value
                              100;
```